tests/sqllogictests/suites/mode/cluster/explain_v2.test

```diff
@@ -428,20 +428,20 @@ query T
 EXPLAIN SELECT a.cluster_node, b.query_node FROM (SELECT name as cluster_node FROM system.clusters) AS a LEFT JOIN (SELECT DISTINCT node_id as query_node FROM system.query_log) AS b ON a.cluster_node = b.query_node
 ----
 Exchange
-├── output columns: [clusters.name (#0), query_node (#62)]
+├── output columns: [clusters.name (#0), query_node (#63)]
 ├── exchange type: Merge
 └── HashJoin
-    ├── output columns: [clusters.name (#0), query_node (#62)]
+    ├── output columns: [clusters.name (#0), query_node (#63)]
```

Comment:
This test why changed?

Reply:
Add new column query tag. So index changed.